client/views/monitorings/list_birds.html

```diff
@@ -176,6 +177,9 @@ <h1 class="page-header">
         <strong>{{row.getSpecies().label.la}}</strong>
       </td>
       <td>{{::row.getCount($language)}}</td>
+      <td>
+        <i ng-if="row.pictures && row.pictures.length > 0" class="fa fa-image fa-fw text-info" preview-pictures="row.pictures"></i>
```

Comment:
The line is too long (also in the other templates). This also is replicated on all forms, so it might be extracted as a partial template and just included, or wrapped in a directive